CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
+
+## [Unreleased]
+
+### Added 
+
+- Add `Enum contains network errors` - [#5](https://github.com/ios-course/simple-network-service/issues/5)
```

Comment:
Provide a link to the PR, not the issue.

Reply:
Thanks for your attention! Fixed it.